projects/adventure/adv.py

```diff
@@ -22,7 +22,56 @@
 player = Player("Name", world.startingRoom)
 
 # Fill this out
+
+'''
+- record traversed rooms
+- add current room
+- 
+- make sure on way back to room you dont have to go retrace steps to go back into each room to get to starter
+as that would add a lot of unneccesary moves (brady)
+'''
+
+
+def oppositeDirection(direction):
```

Comment:
nice job defining a function for this.  Similarly we could do the same with a hash-table/ dictionary.